### ISSUE TITLE
Crash on opening dev-tools. Win7, x64, v2.3.1

### ISSUE THREAD

**Issue by zxqfox** [gitterHQ/desktop#42]
How to repeat:
- Press on "Developer Tools" in menu.
- Crash.

What happens:
- Window with "node-webkit" caption opened without any data and with white content;
- App instantly crashing, and I see these errors in logs:
  ![image](https://cloud.githubusercontent.com/assets/677518/6619972/f61e9e28-c8de-11e4-8ea1-23af01e57a54.png)

**Comment by trevorah** [comment]
Yep, I can reproduce this on 2.3.1, but not on 2.3.0 for some reason...

**Comment by zxqfox** [comment]
+1, I didn't see this in previous release.

**Comment by trevorah** [comment]
judging from the diff, it looks like something to do with the websocket fixes, but I have no idea why: https://github.com/gitterHQ/desktop/compare/2.3.0...2.3.1

I think @suprememoocow might find this quite interesting.

**Comment by MadLittleMods** [comment]
Can you still reproduce this with v3.0.3?
